src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,75 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    private static final String NAME_SUPPLY = "supply";
+    private static final String NAME_BUY = "buy";
+    private static final String NAME_RESULT = "result";
+
     public void getStatistic(String fromFileName, String toFileName) {
+        File file = new File(toFileName);
+        if (file.exists()) {
+            return;
+        }
```

Comment:
What if the content of the file `fromFileName ` will change? Or what if the file is empty? Let's better clear the file before writing new data.